chainer/functions/noise/simplified_dropconnect.py

```diff
@@ -136,6 +142,9 @@ def simplified_dropconnect(x, W, b=None, ratio=.5, train=True, mask=None):
             Otherwise, The mask must be ``(n, M, N)`` shaped array.
```

Comment:
So, users can provide different mask by putting a ndarray of shape `(n, M, N)` to `mask` even if `use_backwise_mask` is true, can't you? I would like to confirm that this specification is on purpose. Even so, I think it is better to check the shape of `mask` if it is broadcastable. You can do it with e.g. [`xp.broadcast_to`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.broadcast_to.html)